drivers/counter/counter_mcux_lpc_rtc.c

```diff
@@ -199,7 +207,9 @@ static void mcux_lpc_rtc_isr(const struct device *dev)
 	if (RTC_GetStatusFlags(config->base) & RTC_CTRL_ALARM1HZ_MASK) {
 		RTC_ClearStatusFlags(config->base, kRTC_AlarmFlag);
 	}
-	RTC_StartTimer(config->base);
+	if (enable) {
+		RTC_StartTimer(config->base);
```

Comment:
same comment as previous ones

Reply:
Thanks for catching this. Update to include this change.